pkg/model/event.go

```diff
@@ -50,3 +50,16 @@ func MakeEventKey(name string, labels map[string]string) string {
 	h.Write([]byte(b.String()))
 	return hex.EncodeToString(h.Sum(nil))
 }
+
+func (e *Event) Filename() string {
+	// TODO: Think about pattern of storing object under {piped-id}_{event_id}.json name.
+	return fmt.Sprintf("%s.json", e.Id)
+}
+
+func (e *Event) DivideToMulti() (bool, []string) {
+	return false, nil
+}
+
+func (e *Event) ColdStorable() bool {
+	return true
+}
```

Comment:
I'm curious about how `ColdStorable()` will be used. According to the current spec, an Event model is likely to be updated until it gets handled.

I'm not sure but shouldn't it be like this? (sorry, this Handled method will be added once https://github.com/pipe-cd/pipecd/pull/3065 got merged)

```suggestion
func (e *Event) ColdStorable() bool {
	return e.Handled()
}
```

Reply:
I get your point. In my imagation, this `ColdStorable` is used to confirm should we process cold storage to find object of this kind or not, not just to control do we should make the current instance to cold storage or not. Which mean (for instance) we will call this `ColdStorable` first when implement cold storage migration logic, then check for one specified instance for it's handled or not by your suggestion `IsHandled`. Besides, this current implementation for the interface can be count as a mock, we can change the implementation later as your suggestion based on the logic we use to build cold storage migration.

Reply:
That makes more sense 👍